test/core/Particle.spec.js

```diff
@@ -317,4 +320,25 @@ describe('particle update', () => {
 
     done();
   });
+
+  it('should have the same value in Particle.velocity if the total time of particles is the same', done => {
```

Comment:
Thanks for the test!

I assume that if I add this test to `master` now, it would fail. Is that correct?